introduction.md

```diff
@@ -2,4 +2,9 @@
 Hello there! My name is Nath, on my hard way to become a github master.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?